RecoJets/JetProducers/src/PileupJetIdAlgo.cc

```diff
@@ -491,7 +522,7 @@ PileupJetIdentifier PileupJetIdAlgo::computeIdVariables(const reco::Jet* jet,
         double dZ0 = 9999.;
         double dZ_tmp = 9999.;
         for (unsigned vtx_i = 0; vtx_i < allvtx.size(); vtx_i++) {
-          const auto& iv = allvtx[vtx_i];
+          auto iv = allvtx[vtx_i];
```

Comment:
Comparing the 12_6 backport with this PR, I notice that this updates was not backported.
While I agree about not backporting it, I wonder why it was even included in the master PR: should it get reverted at some point?